AllReadyApp/Web-App/AllReady/wwwroot/css/site.css

```diff
@@ -49,11 +49,15 @@ textarea {
     padding-right: 40px;
 }
 
-a:not(.unvolunteer,.btn) {
+a:not(.unvolunteer) {
     text-decoration: none;
     color: #ab6627;
 }
 
+a:not(.btn) {

```

Comment:
These could be combined to a single rule which would avoid duplicating the values:
```
a:not(.unvolunteer),
a:not(.btn) {
    text-decoration: none; 
     color: #ab6627; 
}
```